x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TransformStatsTests.java

```diff
@@ -69,7 +69,7 @@ public void testBwcWith73() throws IOException {
                 STARTED,
                 randomBoolean() ? null : randomAlphaOfLength(100),
                 randomBoolean() ? null : NodeAttributeTests.randomNodeAttributes(),
-                new TransformIndexerStats(1, 2, 3, 4, 5, 6, 7, 8, 9, 10),
+                new TransformIndexerStats(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 0.0, 0.0, 0.0),
```

Comment:
do not get confused, `testBwcWith73`: this test BWC with 7.3 where we had no exponential averages, so its ok to construct the stats with `0.0`